django/db/migrations/questioner.py

```diff
@@ -214,6 +219,30 @@ def ask_not_null_alteration(self, field_name, model_name):
                 return self._ask_default()
         return None
 
+    def ask_not_null_removal(self, field_name, model_name):
+        """Removing a NOT NULL field from a model"""
+        if not self.dry_run:
+            choice = self._choice_input(
+                f"It is not recommended to remove a non-nullable field '{field_name}' "
+                f"from '{model_name}' without a default; "
+                f"This is because the database needs something to populate existing "
+                f"rows when migrating backwards.\n"
+                f"Please select a fix:",
+                [
+                    "Provide a one-off default now (will be set on all existing rows).",
```

Comment:
We probably should also ask about setting a database default (`db_default`) here.

Reply:
@LilyFoote hey, you did great work implementing `db_defaults`. 👏  
I think it won't be possible(or tricky) to ask for both `default` and `db_default` in a single prompt. Since the value returned by questioner will be assigned to variable [`default`](https://github.com/django/django/pull/16890/files#diff-50cc08301fdd233fd64196aa90bc684d96ef557d99af67692995559646de567aR1107) only.🤔 
Would really appreciate any suggestions from your side.

Reply:
One thing we could try is return a `(default, db_default)` tuple.

Reply:
> One thing we could try is return a `(default, db_default)` tuple.

🤔 Then maybe we can ask for `db_default` in other methods like `ask_not_null_addition` , `ask_not_null_alteration` too? Lets see what others have to say on this. Till then, i'll try the tuple approach.

Reply:
Yeah, maybe it's worth doing as a separate PR too, so this one can land.

Reply:
Thinking about it a bit more after seeing https://code.djangoproject.com/ticket/34637 is closed, I think I was confusing myself.

I think the only place we need to take `db_default` into account is to make sure reversing a `RemoveField` operation re-adds a `db_default` if one was set.

Reply:
>I think the only place we need to take db_default into account is to make sure reversing a RemoveField operation re-adds a db_default if one was set.

That already works fine as per my testing.👍